docs/guides/http-client.md

```diff
@@ -81,3 +81,15 @@ For the shortcut methods (since the only mandatory option is intrinsic in the fu
 ```ts
 client.get('https://google.it', { validateRequest: false }).then(response => console.log(response));
 ```
+
+You can also use relative links when doing requests. In such case you won't be able to use the proxy and the server validation will be disabled:
```

Comment:
are these relative or root-relative? https://mor10.com/html-basics-hyperlink-syntax-absolute-relative-and-root-relative/

Reply:
Root relative. `/pet/10`